src/components/Identity/createComponent.js

```diff
@@ -36,9 +36,9 @@ export default ({
       }
     },
     commands: {
-      setCustomerIds: {
+      syncIdentity: {
         run: options => {
-          return customerIds.sync(options);
+          return userIds.sync(options.userIds);
```

Comment:
We should add some validation so we don't get an error if the user passes the wrong key.
```
objectOf({userIds: objectOf({}).required()}).noUnknownFields().required()
```